### PR TITLE
Update Cilium v1.15.5

### PR DESCRIPTION
**Pull request by kvaps**
This PR updates Cilium from 14 to 15

### CODE REVIEW

File: packages/system/cilium/charts/cilium/Chart.yaml [hunk 1]
```diff
@@ -1,131 +1,88 @@
...
+    \ version: v2alpha1\n  name: ciliumpodippools.cilium.io\n  displayName: Cilium
+    Pod IP Pool\n  description: |\n    CiliumPodIPPool defines an IP pool that can
+    be used for pooled IPAM (i.e. the multi-pool IPAM mode).\n"
 apiVersion: v2
-appVersion: 1.14.10
+appVersion: 1.15.5
 description: eBPF-based Networking, Security, and Observability
 home: https://cilium.io/
-icon: https://cdn.jsdelivr.net/gh/cilium/cilium@v1.14/Documentation/images/logo-solo.svg
+icon: https://cdn.jsdelivr.net/gh/cilium/cilium@main/Documentation/images/logo-solo.svg
```
Comment (themoriarti) [inline packages/system/cilium/charts/cilium/Chart.yaml:85]: main can change, why not specify the exact version?

**Review by themoriarti — COMMENTED** [review]
This is a very big change, has functional testing or smoke testing been done on the main features, i.e. all deletion and editing objects have been implemented?

I've generally gone through the changes and everything looks good.

File: packages/system/cilium/charts/cilium/Chart.yaml (hunk 1 above)
Comment (kvaps) [reply packages/system/cilium/charts/cilium/Chart.yaml:85]: It's imported code, this value is not used anywhere

**Review by themoriarti — APPROVED** [review]
LGTM

**Pull request merged**